html/infrastructure/safe-passing-of-structured-data/transfer-errors.window.js

```diff
@@ -0,0 +1,27 @@
+function assert_transfer_error(transferList) {
+  assert_throws("DATA_CLONE_ERR", () => self.postMessage({ get whatever() { throw 1 } }, "*", transferList));
```

Comment:
nit: DataCloneError (at least http://web-platform-tests.org/writing-tests/testharness-api.html says the _ERR form is only supported "for compatibility with existing tests")